tests/conftest.py

```diff
@@ -215,3 +215,23 @@ def df_normal_dist():
     df.columns = ["var"]
 
     return df
+
+
+@pytest.fixture(scope="module")
+def df_enc_categorical_and_numeric():
```

Comment:
can we use df_vartypes instead of creating a new one? is this just to check the variable selection?